test/suite/test_calc_modify.py

```diff
@@ -83,10 +83,10 @@ def one_test(self, c, k, oldsz, repeatsz, nmod, maxdiff):
         orig = oldv
         newv = '' if self.valuefmt == 'S' else b''
         for i in range(nmod):
-            if i > 0 and offsets[i] - offsets[i - 1] < maxdiff:
+            if i > 0 or offsets[i] - offsets[i - 1] < maxdiff:
```

Comment:
Should the _and_ be changed to _or_? it seems like if any value _i > 0_ will cause the continue branch constantly. Not sure if that was intended?

Reply:
You're right, the fix should have been making i > 1. Thanks!